### ISSUE TITLE
Serialize the warmed up torchscript module

### ISSUE THREAD

**Issue by Dango233** [pytorch/pytorch#84615]
### 🚀 The feature, motivation and pitch

- JIT and torchscript can provide a huge performance boost for models, but the long warm-up time for large models like stable diffusion unet(~90s) can be a huge blocker for production, even individual use
- If there could be a way of serializing the warmed-up module (even with limitation that needs to be run on specific hardware), would be super helpful

### Alternatives

- Tried to `torch._C._jit_set_profiling_executor(False) eliminates the extra recompilations` but this eliminates performance gain

**Comment by davidberard98** [comment]
As a workaround for the long warmup time, there might be some other possible solutions:
- Depending on what pytorch version you're on, you could try using a different fuser. https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/OVERVIEW.md#fusers
- If you're using NNC, you can change the fusion strategy to use dynamic fusion directly, if respecialization on static shapes is causing performance issues. See https://pytorch.org/docs/stable/generated/torch.jit.set_fusion_strategy.html.

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
